pawno/include/PPC_PlayerCommands.inc

```diff
@@ -2824,14 +2824,18 @@ COMMAND:stats(playerid, params[])
 		if(sscanf(params, "U(-1)", StatsPlayer)) SendClientMessage(playerid, COLOR_RED, "Usage: \"/stats <StatsPlayer (optional)>\"");
 		else
 		{
-			if (INVALID_PLAYER_ID == StatsPlayer)
+			if (-1 == StatsPlayer)
```

Comment:
This is the only change which is needed. The rest can stay like it was before. I have tested it and it worked for me. I could not test it with multiple players but it should work properly.